static/js/project/controllers/project.controller.js

```diff
@@ -194,10 +212,10 @@
       }
       function getStepName(stepId){
           if(stepId==1){
-              return '1. Category';
+              return '1. Getting Started';
```

Comment:
no sure about this, these are definitely categories so it can stay as is, let's worry about renaming later